app/models/client.rb

```diff
@@ -26,6 +26,8 @@
 #  fk_rails_...  (vita_partner_id => vita_partners.id)
 #
 class Client < ApplicationRecord
+  devise :lockable, :timeoutable, :trackable
```

Comment:
On the secret number page, let's lock after 5 attempts, with no unlock policy (since Devise an auto-unlock by time or by sending an email confirmation link; let's do neither).

I think there a method to `unlock` a `lockable`, but I don't _really_ know.

If they're locked, we should a message saying they should contact support. Support should file a chore for eng to contact the client, find out if it's really them, and then ask for eng to unlock the account.